diff --git a/plugins/multimedia/gstreamer/gstreamer.pro b/plugins/multimedia/gstreamer/gstreamer.pro
index bc5bf23..5865ecd 100644
--- a/plugins/multimedia/gstreamer/gstreamer.pro
+++ b/plugins/multimedia/gstreamer/gstreamer.pro
@@ -70,5 +70,5 @@ contains(QT_CONFIG, multimedia) {
 include(mediaplayer/mediaplayer.pri)
 include(mediacapture/mediacapture.pri)
 
-target.path=$$QT_MOBILITY_PREFIX/plugins/mediaservice
+target.path=$$QT_MOBILITY_LIB/plugins/mediaservice
 INSTALLS+=target
diff --git a/plugins/multimedia/m3u/m3u.pro b/plugins/multimedia/m3u/m3u.pro
index d24fbea..8f9dcd2 100644
--- a/plugins/multimedia/m3u/m3u.pro
+++ b/plugins/multimedia/m3u/m3u.pro
@@ -21,5 +21,5 @@ symbian {
     TARGET.EPOCALLOWDLLDATA = 1
 }
 
-target.path=$$QT_MOBILITY_PREFIX/plugins/playlistformats
+target.path=$$QT_MOBILITY_LIB/plugins/playlistformats
 INSTALLS+=target
diff --git a/plugins/multimedia/phonon/phonon.pro b/plugins/multimedia/phonon/phonon.pro
index aee2c16..bd6709d 100644
--- a/plugins/multimedia/phonon/phonon.pro
+++ b/plugins/multimedia/phonon/phonon.pro
@@ -32,5 +32,5 @@ symbian {
     TARGET.CAPABILITY = ALL -TCB
     TARGET.EPOCALLOWDLLDATA = 1
 }
-target.path=$$QT_MOBILITY_PREFIX/plugins/mediaservice
+target.path=$$QT_MOBILITY_LIB/plugins/mediaservice
 INSTALLS+=target
diff --git a/plugins/multimedia/pulseaudio/pulseaudio.pro b/plugins/multimedia/pulseaudio/pulseaudio.pro
index efb9f33..1d6cade 100644
--- a/plugins/multimedia/pulseaudio/pulseaudio.pro
+++ b/plugins/multimedia/pulseaudio/pulseaudio.pro
@@ -23,5 +23,5 @@ SOURCES += wavedecoder.cpp \
     pulseaudioplayerservice.cpp \
     pulseaudioplayercontrol.cpp
 
-target.path = $$QT_MOBILITY_PREFIX/plugins/mediaservice
+target.path = $$QT_MOBILITY_LIB/plugins/mediaservice
 INSTALLS += target
diff --git a/plugins/multimedia/v4l/v4l.pro b/plugins/multimedia/v4l/v4l.pro
index 17a7fa0..c05a167 100644
--- a/plugins/multimedia/v4l/v4l.pro
+++ b/plugins/multimedia/v4l/v4l.pro
@@ -19,5 +19,5 @@ SOURCES += v4lserviceplugin.cpp
 include(radio/radio.pri)
 include(camera/camera.pri)
 
-target.path=$$QT_MOBILITY_PREFIX/plugins/mediaservice
+target.path=$$QT_MOBILITY_LIB/plugins/mediaservice
 INSTALLS+=target
diff --git a/plugins/sensors/dummy/dummy.pro b/plugins/sensors/dummy/dummy.pro
index a093a4d..ff24a25 100644
--- a/plugins/sensors/dummy/dummy.pro
+++ b/plugins/sensors/dummy/dummy.pro
@@ -19,7 +19,6 @@ equals(STRICT,1) {
     QMAKE_LFLAGS+=-Wl,-no-undefined
 }
 
-DESTDIR = $$OUTPUT_DIR/bin/examples/sensors
-target.path = $$SOURCE_DIR/plugins/sensors
+target.path = $$QT_MOBILITY_LIB/plugins/sensors 
 INSTALLS += target
 
diff --git a/plugins/sensors/generic/generic.pro b/plugins/sensors/generic/generic.pro
index b126b5f..8e2c647 100644
--- a/plugins/sensors/generic/generic.pro
+++ b/plugins/sensors/generic/generic.pro
@@ -19,7 +19,6 @@ equals(STRICT,1) {
     QMAKE_LFLAGS+=-Wl,-no-undefined
 }
 
-DESTDIR = $$OUTPUT_DIR/bin/examples/sensors
-target.path = $$SOURCE_DIR/plugins/sensors
+target.path = $$QT_MOBILITY_LIB/plugins/sensors
 INSTALLS += target
 
diff --git a/plugins/sensors/n900/n900.pro b/plugins/sensors/n900/n900.pro
index 9e7716e..a65ad60 100644
--- a/plugins/sensors/n900/n900.pro
+++ b/plugins/sensors/n900/n900.pro
@@ -18,7 +18,6 @@ equals(STRICT,1) {
     QMAKE_LFLAGS+=-Wl,-no-undefined
 }
 
-DESTDIR = $$OUTPUT_DIR/bin/examples/sensors
-target.path = $$SOURCE_DIR/plugins/sensors
+target.path = $$QT_MOBILITY_LIB/plugins/sensors
 INSTALLS += target
 
